cmd/config/get_orgs_config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"github.com/Shopify/voucher/repository"
+	"github.com/spf13/viper"
+)
+
+func GetOrganizationsFromConfig() (orgs map[string]repository.Organization) {
+	orgs = make(map[string]repository.Organization)
+	repositories, ok := viper.Get("repositories").([]interface{})
+	if !ok {
+		return
```

Comment:
We should add some logging here to inform the user that there are no repositories if there are none.

We could also simplify this function by replacing this line with the following:

```suggestion
		repositories = []interface{}{}
```

This will mean that the loop on line 14 will not execute, and we can then log if `orgs` is empty.

Reply:
Done